lib/config/flat-config-schema.js

```diff
@@ -119,7 +129,7 @@ function normalizeRuleOptions(ruleOptions) {
         : [ruleOptions];
 
     finalOptions[0] = ruleSeverities.get(finalOptions[0]);
-    return finalOptions;
+    return finalOptions.map(value => structuredClone(value));
```

Comment:
```suggestion
    return structuredClone(finalOptions);
```

We could simplify it this way?